SMLHelper/MonoBehaviours/Fixer.cs

```diff
@@ -7,61 +7,35 @@ namespace SMLHelper.V2.MonoBehaviours
     /// <summary>
     /// This <see cref="MonoBehaviour"/> is <c>automatically</c> added to any <see cref="GameObject"/> created through <see cref="ModPrefab.GetGameObject"/>.
     /// </summary>
-    public class Fixer : MonoBehaviour, IProtoEventListener
+    internal class Fixer : MonoBehaviour
```

Comment:
Since this has been `public` for so long, it's hard to know for sure if there aren't mods out there that are adding their own `Fixer` components manually.

For the sake of not breaking anything, let's keep it `public` just to be safe.  
There's no harm in allowing this to be `public` anyways.